Sources/Testing/Testing.docc/LimitingExecutionTime.md

```diff
@@ -23,7 +23,7 @@ complete effectively, consider setting a time limit for it so that it's marked a
 ``Trait/timeLimit(_:)`` trait as an upper bound:
 
 ```swift
-@Test(.timeLimit(.seconds(60 * 60))
+@Test(.timeLimit(.minutes(60))
```

Comment:
Should we use `.minutes(1)`? Or something a little shorter than 60 minutes?

Reply:
This example is specifically using "one hour". I'd be impressed if we served that many customers in one minute! :)